src/modules/fancyzones/editor/FancyZonesEditor/StringUtils.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+
+namespace FancyZonesEditor.Utils
+{
+    public static class StringUtils
+    {
+        public static string ToDashCase(this string str)
```

Comment:
Maybe we could rename it to `camelCaseToDashCaser` or something like that, to indicate input and output case style ?

Reply:
Done